lib/diagnostic.rb

```diff
@@ -5,36 +5,37 @@ class Diagnostic
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max(&:size)
```

Comment:
`max` is expecting a block telling it how to compare two entries, using the 'spaceship' operator. 
i.e. `enum.max { |a, b| a.size <=> b.size }`